Data/src/main/java/org/tribuo/data/columnar/processors/response/QuartileResponseProcessor.java

```diff
@@ -101,9 +166,42 @@ public String getFieldName() {
         return Optional.of(output);
     }
 
+    @Override
+    public Optional<T> process(List<String> values) {
+        if(values.size() != fieldNames.size()) {
+            throw new IllegalArgumentException("values must have the same length as fieldNames. Got values: " + values.size() + " fieldNames: "  + fieldNames.size());
+        }
+        List<String> response = new ArrayList<>();
+        for(int i=0; i< fieldNames.size(); i++) {
+            String value = values.get(i);
+            String prefix = name == null || name.isEmpty() ? fieldNames.get(i) : getFieldName();
```

Comment:
Shouldn't the last part of the ternary be `: name`? Otherwise this has different behaviour depending on what process method you call.